src/main/java/edu/harvard/iq/dataverse/DatasetFieldServiceBean.java

```diff
@@ -717,6 +733,9 @@ private JsonObject filterResponse(JsonObject cvocEntry, JsonObject readObject, S
                 }
             }
         }
+        if(nrOfNotFound>0) {
+            logger.warning("External Vocabulary: " + termUri + " - Failed to find value(s) reported above in " +readObject);
```

Comment:
See comment https://github.com/DANS-KNAW-jp/dataverse/blob/5e59050a492ba2641e63a441d962ffe63ba05925/src/main/java/edu/harvard/iq/dataverse/DatasetFieldServiceBean.java#L742 
When should we also return null, for example, should we at least have a termName?

Reply:
Probably a matter of opinion - if we send partial responses, all code using this has to be ready for missing fields. If that's OK, I guess I'd send any fields that match the filter rather than deciding that termName is more important.